hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -47,6 +51,10 @@ public static HawkBuilder init(Context context) {
       return remove(key);
     }
 
+    if (!isHawkInitialised()) {
+      return false;
```

Comment:
the annotation IntDef is included in the support library. Hawk does not depend to it.